test_data/asm/full_pil_constant.asm

```diff
@@ -1,5 +1,5 @@
 machine FullConstant {
-	degree 2;
+	degree 4;
```

Comment:
I think degree 2 is not supported anymore because now we need `degree + 1` values, which means that there is only one row left for blinding factors which is not enough.

We could handle that case, but this should not be a restriction for any practical purpose (I guess for the same reason we previously didn't support a degree of 1).

Reply:
yea makes sense